articles/azure-cache-for-redis/cache-configure.md

```diff
@@ -461,6 +461,7 @@ For more information about databases, see [What are Redis databases?](cache-deve
 > * SAVE
 > * SHUTDOWN
 > * SLAVEOF
+> * REPLICAOF
```

Comment:
ACL is also not supported. Both ACL and REPLICAOF were added as part of Redis 6 release.

Reply:
Thanks @MECHANDR. I've added a commit to add that commnad, too.